addons/project/static/src/components/project_task_state_selection/project_task_state_selection.js

```diff
@@ -77,7 +77,8 @@ export class ProjectTaskStateSelection extends StateSelectionField {
             ["1_canceled", _t("Canceled")],
             ["1_done", _t("Done")],
         ];
-        if (this.currentValue != "04_waiting_normal") {
+        const currentState = this.props.record.data[this.props.name]
```

Comment:
You forgot `;` in your keyboard. 😇 

```suggestion
        const currentState = this.props.record.data[this.props.name];
```